packages/components/src/Components/SkeletonTable/SkeletonTable.test.js

```diff
@@ -18,4 +18,25 @@ describe('SkeletonTable component', () => {
         const wrapper = shallow(<SkeletonTable columns={[ 'first', 'second' ]} rowSize={15} />);
         expect(toJson(wrapper)).toMatchSnapshot();
     });
+
+    it.skip('should render correctly with checkboxes', () => {
+        const wrapper = shallow(<SkeletonTable
+            columns={[ 'first', 'second' ]}
+            rowSize={15}
+            isSelectable={ true }
+        />);
+
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it.skip('should render correctly with radio buttons', () => {
```

Comment:
No need to skip these
```suggestion
    it('should render correctly with radio buttons', () => {
```